src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -100,6 +100,7 @@ namespace Akka.Actor
         public void ReserveChild(string name) { }
         public void Restart(System.Exception cause) { }
         public void Resume(System.Exception causedByFailure) { }
+        public void SendSystemMessage(Akka.Dispatch.SysMsg.ISystemMessage systemMessage) { }
```

Comment:
Added overload of `SendSystemMessage` which doesn't take a `Sender`, since it's not relevant, to be consistent with the JVM.